src/pages/commits/index.tsx

```diff
@@ -87,6 +90,7 @@ const Commits = () => {
   const statusFilters = toArray(parsed[ProjectFilterOptions.Status]);
   const variantFilters = toArray(parsed[ProjectFilterOptions.BuildVariant]);
   const taskFilters = toArray(parsed[ProjectFilterOptions.Task]);
+  const viewFilter = parsed[ProjectFilterOptions.View] as ProjectHealthView;
```

Comment:
Can we use the useQueryParam hook here? I would like to eventually replace these all with that hook.